docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -16,17 +16,19 @@
   <<dfanalytics-types>>.
   
 `analyzed_fields`::
-  (object) You can specify both `includes` and/or `excludes` patterns. If 
-  `analyzed_fields` is not set, only the relevant fields will be included. For 
-  example, all the numeric fields for {oldetection}. For the supported field 
-  types, see <<ml-put-dfanalytics-supported-fields>>.
+  (Optional, sobject) Specify `includes` and/or `excludes` patterns to select
+  which fields will be included in the analysis. If `analyzed_fields` is not set,
+  only the relevant fields will be included. For example, all the numeric fields
+  for {oldetection}. For the supported field types, see <<ml-put-dfanalytics-supported-fields>>.
+  Also see {ref}/explain-dfanalytics.html[_explain API] which helps understand
```

Comment:
Should it be "Also see _the_ \_explain API" rather than just "Also see \_explain API"?